gandalf.js

```diff
@@ -3,8 +3,8 @@
  *
```

Comment:
SCRIPT-I'm a bit concerned about the potential performance impact of this loop. Have you thought about optimizing it?